src/server/command/worldedit/copy.zig

```diff
@@ -0,0 +1,44 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+
+pub const description = "Copy selection to clipboard.";
+pub const usage = "/copy";
+
+pub fn execute(args: []const u8, source: *User) void {
+	if(args.len != 0) {
+		source.sendMessage("#ff0000Too many arguments for command /copy. Expected no arguments.", .{});
+		return;
+	}
+
+	if(source.worldEditData.selectionPosition1) |pos1| {
```

Comment:
It's generally more readable to use early returns instead of nested if elses:
```zig
const pos1 = source.worldEditData.selectionPosition1 orelse {
	return source.sendMessage("#ff0000Position 1 isn't set", .{});
};
const pos2 = source.worldEditData.selectionPosition2 orelse {
	return source.sendMessage("#ff0000Position 2 isn't set", .{});
};
```